l10n/en/brands.ftl

```diff
@@ -155,6 +155,7 @@
 -brand-name-mac = macOS
 -brand-name-mac-short = Mac
 -brand-name-ubuntu = Ubuntu
+-brand-name-debian = Debian
```

Comment:
I think we can remove this as we no longer use brand name variables for third parties